pyspark_ai/pyspark_ai.py

```diff
@@ -5,7 +5,13 @@
 from typing import Callable, List, Optional
 from urllib.parse import urlparse
 
-import pandas as pd  # noqa: F401
+# We only try to import pandas here to circumvent certain issues
+# when the generated code assumes that pandas needs to be imported
+# before actually evaluating the code.
+try:
```

Comment:
Shall we move this block under method `plot_df`?

Reply:
I think this is the tricky part here, Should we simply use the same pandas requirement as defined by Spark? >1.0.5?

Reply:
https://github.com/apache/spark/blob/master/python/setup.py#L133

Essentially, we create an incompatible package to PySpark otherwise.

Reply:
Created https://github.com/databrickslabs/pyspark-ai/pull/93 for this.